@navikt/core/css/form/combobox.css

```diff
@@ -0,0 +1,212 @@
+.navds-combobox__wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  position: relative;
+}
+
+.navds-combobox__wrapper-inner {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  border: 1px solid black;
+  padding: 4px;
+  padding-right: calc(32px + 2.5rem);
+}
+
+.navds-combobox__wrapper-inner:hover {
+  cursor: text;
+}
+
+.navds-combobox__selected-options {
+  gap: var(--a-spacing-2);
+}
+
+.navds-combobox__selected-options > li {
+  margin: auto 0;
+}
+
+.navds-combobox__selected-options > li:last-of-type {
+  display: flex;
+  flex: 1;
+}
+
+.navds-combobox__input-wrapper {
+  width: 100%;
+}
+
+.navds-combobox__input {
+  z-index: 1;
+  flex: 1;
+  border: none;
+  padding: 0;
+  padding-right: var(--a-spacing-10);
+  margin: 0;
+}
+
+.navds-combobox__input:focus {
+  outline: none;
+  border: none;
+  box-shadow: none;
+}
+
+.navds-combobox__wrapper-inner:has(.navds-combobox__input:focus) {
+  box-shadow: 0 0 0 1px var(--a-surface-default) inset, var(--a-shadow-focus);
+  box-shadow: var(--a-shadow-focus);
+}
+
+.navds-combobox__wrapper-inner:has(.navds-combobox__input:focus).navds-combobox__wrapper-inner--virtually-unfocused {
+  box-shadow: none;
+}
```

Comment:
Kan endres til `focus-visible` med fallback til `:focus` for ikke støttede browsere (se button.css)

Reply:
Done, men usikker på om det gir verdi. Ser ikke noen forskjell, og CSS-en blir dobbelt så stor.
Er ikke focus-visible mest for knapper, o.l.?